events/wc_events/wc_lordaeron_events.txt

```diff
@@ -0,0 +1,410 @@
+				modifier = { add = 5 root = { piety_level >= 3 } }
+				modifier = { add = 5 root = { piety_level >= 4 } }
+				modifier = { add = 5 root = { learning > decent_skill_rating }}
+				modifier = { add = 5 root = { dread > 20 } }
+				modifier = { add = 5 root = { dread > 40 } }
+				modifier = { add = 5 root = { dread > 60 } }
+				modifier = { add = -15 root = { dread > 80 } }
+				# modifier for court chaplain piety level or intelligence?
+				title:c_lordaeron = { 
+					set_county_faith = faith:forsaken_cult
+				}
+			}
+			45 = { 
+				title:c_lordaeron = { 
+					set_county_faith = faith:forgotten_shadow
+				}
+			}
+		}
+		add_character_modifier = { 
+			modifier = wc_forsaken_tolerated_selinite_modifier
+		}
+		ai_chance = { 
+			base = 5
+			ai_value_modifier = {
+				ai_honor = 0.5
+				ai_compassion = 1
+				ai_boldness = -1
+				ai_zeal = -1
+			}
+		}
+	}
+}
+
+lordaeron.1102 = { 
+	hidden = yes 
+	type = character_event
+
+	immediate = { 
+
+		root = { save_scope_as = actor }
+
+		primary_title = {
+			every_in_de_facto_hierarchy = {
+				limit = { 
+					AND = { 
+						faith = faith:death_god 
+						tier = tier_county  
+					}
+				}
+				set_county_faith = faith:forsaken_cult 
+			}
+		}
+
+		every_courtier_or_guest = {
+			limit = { 
+				OR = {
+					culture = culture:scourge
+					faith = faith:death_god
+				}
+			}
+			this = { save_temporary_scope_as = recipient }
+			new_faith_created_conversion_effect = yes
+		}
+
+		every_vassal_or_below = { 
+			limit = { 
+				OR = {
+					culture = culture:scourge 
+					faith = faith:death_god
+				}
+			}
+
+			this = { save_temporary_scope_as = recipient }
+			new_faith_created_conversion_effect = yes
+
+			every_courtier_or_guest = {
+				limit = { 
+					OR = {
+						culture = culture:scourge
+						faith = faith:death_god
+					}
+				}
+				this = { save_temporary_scope_as = recipient }
+				new_faith_created_conversion_effect = yes
+			}
+		}
+	}
+}
+
+lordaeron.1103 = { 
+	hidden = yes 
+	type = character_event
+
+	immediate = { 
+		root = { 
+			this = { save_temporary_scope_as = recipient }
+			new_faith_created_conversion_effect = yes
+		}
+
+		root ={ save_scope_as = actor }
+
+		every_courtier_or_guest = {
+			limit = { 
+				OR = {
+					culture = culture:forsaken 
+					faith = faith:forsaken_cult
+					faith = faith:death_god
+				}
+			}
+			this = { save_temporary_scope_as = recipient }
+			new_faith_created_conversion_effect = yes
+		}
+
+		every_vassal_or_below = { 
+			limit = { 
+				OR = {
+					culture = culture:forsaken 
+					faith = faith:forsaken_cult
+					faith = faith:death_god
+				}
+			}
+			this = { save_temporary_scope_as = recipient }
+			new_faith_created_conversion_effect = yes
+
+			every_courtier_or_guest = {
+				limit = { 
+					OR = {
+						culture = culture:forsaken 
+						faith = faith:forsaken_cult
+						faith = faith:death_god
+					}
+				}
+				this = { save_temporary_scope_as = recipient }
```

Comment:
```suggestion
				save_temporary_scope_as = recipient
```

Reply:
fixed these and the ones you didn't mark too lol, i know there were a lot. coming from statically typed languages i get really nervous about scope lol